src/classes/HH_OppContactRoles_TEST.cls

```diff
@@ -296,97 +284,84 @@ private class HH_OppContactRoles_TEST {
+        con = getContacts(con.id)[0];
         
-        Id householdId = createdContacts[0].npo02__household__c;
+        Id householdId = con.npo02__household__c;
         
-        //create opportunity
-        Opportunity o = new Opportunity(
-            Name='test oppty',
-            CloseDate=date.newInstance(2001,12,12),
-            StageName=UTIL_UnitTestData_TEST.getClosedWonStage(),
-            Amount=100,
-            AccountId=accountId
-        );
+        Opportunity o = buildOpportunity(accountId);
         Test.startTest();
         insert o;
         Test.stopTest();
-        //check for contact roles
-        List<OpportunityContactRole> createdRoles = [select Role, ContactId, IsPrimary from OpportunityContactRole where OpportunityId = :o.id order by ContactId];
-        system.AssertEquals(0,createdRoles.size());
+
+        List<OpportunityContactRole> createdRoles = getOppContactRoles(o.id);
+        System.assertEquals(0,createdRoles.size());
     }
     
-    private static testmethod void ghIssue25() {
-        npe01__Contacts_and_Orgs_Settings__c contactSettingsForTests = UTIL_CustomSettingsFacade.getContactsSettingsForTests(
-            new npe01__Contacts_and_Orgs_Settings__c (
+    @isTest
+    private static void ghIssue25() {
+        UTIL_CustomSettingsFacade.getContactsSettingsForTests(
+            new npe01__Contacts_and_Orgs_Settings__c(
                 npe01__Enable_Opportunity_Contact_Role_Trigger__c = true,
-                npe01__Opportunity_Contact_Role_Default_role__c = 'Donor',
+                npe01__Opportunity_Contact_Role_Default_role__c = OCR_DONOR_ROLE,
                 npe01__Account_Processor__c = CAO_Constants.ONE_TO_ONE_PROCESSOR));
         
-        npo02__Households_Settings__c householdSettingsForTests = UTIL_CustomSettingsFacade.getHouseholdsSettingsForTests(
+        UTIL_CustomSettingsFacade.getHouseholdsSettingsForTests(
             new npo02__Households_Settings__c ( 
                 npo02__Household_Contact_Roles_On__c = true,
-                npo02__Household_Member_Contact_Role__c = 'Household Member'));
+                npo02__Household_Member_Contact_Role__c = HH_MEMBER_CONTACT_ROLE));
        
         npo02__Household__c hh = new npo02__Household__c(Name ='hh');
         insert hh;
-        Contact con1= new Contact(LastName='test1',npo02__Household__c = hh.id);
+        Contact con1= new Contact(LastName='test1', npo02__Household__c = hh.id);
         Contact con2= new Contact(LastName='test2');
         insert con1;
         insert con2;
 
         Test.startTest();
-        Opportunity o1 = new Opportunity(
+        insert new Opportunity(
             Name = 'o1',
             StageName = UTIL_UnitTestData_TEST.getClosedWonStage(),
             CloseDate = System.today(),
-            Primary_Contact__c = (string)con1.id);
-        insert o1;
-        Opportunity o2 = new Opportunity(
+            Primary_Contact__c = (String)con1.id);
+            
```

Comment:
Looks like a line of spaces is being inserted here.

Reply:
It is not visible in VSCode. Regardless, I re-wrote the statement to use one `insert` instead and removed the line completely. :)